src/markup.js

```diff
@@ -19,7 +19,7 @@ function getBadge(stats, name) {
   const totalCount = stats.total;
   const passedCount = totalCount - failedCount;
 
-  const badgeCountText = failedCount > 0 ? `${`${failedCount}/${totalCount}`}` : `${`${passedCount}/${totalCount}`}`;
+  const badgeCountText = failedCount > 0 ? `${failedCount}/${totalCount}` : `${passedCount}/${totalCount}`;
```

Comment:
It's unclear to me why this nested interpolation might be necessary. From everything I can see in the code, the extra interpolation blocks weren't doing anything, so I removed them.

Reply:
I think it's there because it's added to the markdown and those extra ` are leftover to escape it in the markdown.

Reply:
That doesn't seem to be the case when I run the code in the console:
![Screen Shot 2022-03-16 at 3 22 26 PM](https://user-images.githubusercontent.com/1235376/158693260-b1c81f46-1968-461a-b5f1-fd590590ee7e.png)

Does it need to end up with some other output? If it's not doing what it should do already, I can update it for that!

Reply:
That does make sense...it was probably a typo to begin with then.  Let's go ahead with this change then.  If for some reason the change makes it come out weird (which I don't think it will based on this) then we can put it back in.